gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/transform/SnapshotAggregatorClassVisitor.kt

```diff
@@ -1,8 +1,9 @@
 package com.emergetools.android.gradle.tasks.snapshots.transform
 
-import org.jf.dexlib2.AccessFlags
+import org.gradle.internal.cc.base.logger
```

Comment:
I think this is the wrong import seems like this is a relocated class.
If we want to log stuff here we’ll have to pass in a logger to the constructor.